client/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+	"strings"
+
+	"github.com/mitchellh/go-homedir"
+	pb "github.com/skuid/keyman/shapes"
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+func requestCert(client pb.SignerClient, content []byte, principals []string) ([]byte, error) {
+	request := &pb.SignRequest{
+		Key:        content,
+		Principals: principals,
+	}
+	resp, err := client.Sign(context.Background(), request)
+	if err != nil {
+		return nil, err
+	}
+	return resp.Certificate, nil
+}
+
+func main() {
+	flag.String("server", "localhost:3000", "The server to connect to")
+	flag.StringSlice("principals", []string{"core", "openvpnas"}, "The identities to request")
```

Comment:
i don't know why, but i wish this was `principle`. When doing multiple, it would be `--principle core --principle openvpnas`.

Reply:
your call though.

Reply:
you can also do  `--principals root,core` so I'll just keep it